docs/source/dev-guide/deep-dives/logging.md

```diff
@@ -0,0 +1,84 @@
+(deep_dive_logging)=
+# Logging in Conda
+
+Logging in Conda is based on the [Python logging framework](https://docs.python.org/3/library/logging.html).
```

Comment:
Conda is not capitalized, except for start of sentences and titles. (please update it in the doc)